logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/connection/ConnectionScope.kt

```diff
@@ -1,13 +1,15 @@
 package com.wire.kalium.logic.feature.connection
 
 import com.wire.kalium.logic.data.connection.ConnectionRepository
-import com.wire.kalium.logic.data.user.UserRepository
 
 class ConnectionScope(
     private val connectionRepository: ConnectionRepository
 ) {
 
-    internal val syncConnections: SyncConnectionsUseCase get() = SyncConnectionsUseCaseImpl(
-        connectionRepository = connectionRepository
-    )
+    internal val syncConnections: SyncConnectionsUseCase
+        get() = SyncConnectionsUseCaseImpl(
+            connectionRepository = connectionRepository
+        )
+
+    val sendConnectionRequestUseCase: SendConnectionRequestUseCase get() = SendConnectionRequestUseCaseImpl(connectionRepository)
```

Comment:
To follow what we have been doing in other scope classes, you can remove the `UseCase` part of the val name.

```suggestion
    val sendConnectionRequest: SendConnectionRequestUseCase get() = SendConnectionRequestUseCaseImpl(connectionRepository)
```